src/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualResponseBuilder.java

```diff
@@ -91,6 +91,14 @@ ResponseValues assembleResponse() throws TemplateModelException {
 		}
 		
 		/*
+		//Adding variable: homeCountry (Hao, June 2015)
+		Individual homeCountry = individual.getRelatedIndividual("http://info.deepcarbon.net/schema#homeCountry");
+		if (homeCountry != null){
+			body.put("homeCountry", homeCountry.getRdfsLabel());
+		}
+		*/
+		
+		/*
```

Comment:
We were going to add home country to the top of the profile, but it's already at the top of the Overview section so I decided we didn't need it. But, just in case engagement doesn't agree, for now we can leave it commented out.